grc/lora_receiver.xml

```diff
@@ -4,7 +4,7 @@
   <key>lora_lora_receiver</key>
   <category>[LoRa]</category>
   <import>import lora</import>
-  <make>lora.lora_receiver($in_samp_rate, $freq, $offset, $sf, $out_samp_rate)</make>
+  <make>lora.lora_receiver($in_samp_rate, $freq, $offset, $sf, $out_samp_rate, $threshold)</make>
 
```

Comment:
This change is not reflected in lora_receiver.py, resulting in a runtime TypeError.